plugins/module_utils/job.py

```diff
@@ -316,9 +316,8 @@ def _get_job_status(job_id="*", owner="*", job_name="*", dd_name=None, dd_scan=T
                 try:
                     list_of_dds = jobs.list_dds(entry.job_id)
                 except exceptions.DDQueryException as err:
-                    if 'BGYSC5201E' in str(err):
-                        is_dd_query_exception = True
-                        pass
+                    is_dd_query_exception = True
+                    pass
```

Comment:
The `pass` keyword is not needed since we have another line running inside this block

Reply:
updated it please review